steward/src/config.rs

```diff
@@ -209,7 +209,7 @@ impl Default for GravitySection {
     fn default() -> Self {
         Self {
             contract: "0x0000000000000000000000000000000000000000".to_owned(),
-            fees_denom: "stake".to_owned(),
+            fees_denom: "usomm".to_owned(),
```

Comment:
@hannydevelop I definitely missed this. Next time would you mind pointing it out in a review instead of committing to my branch?

Reply:
yea, no P. I just figured this one commit shouldn't make a PR

Reply:
thanks!